### ISSUE TITLE
[Other]Is this a bug or a UI issue?

### ISSUE THREAD

**Issue by FMYC2015** [BlackBoxing/BlackBox#2]
![360截图16800413337062](https://user-images.githubusercontent.com/91420484/182503062-5d946c8b-d90c-4946-82d5-9c3be672f0ea.jpg)

**Comment by BlackBoxing** [comment]
No. It's caused by the auto generated by android dark theme due to the uncompatibility. If you're android developer, you can refer to the tutorial about compatibility with android dark theme or just disable dark theme of system.
